src/orquestra/integrations/braket/conversions/_circuit_conversions.py

```diff
@@ -20,9 +20,6 @@ def export_to_braket(circuit: Circuit) -> BraketCircuit:
       BraketCircuit
     """
 
-    for i in range(circuit.n_qubits):
```

Comment:
I'm not sure. My suspicion is that it might have its roots in case where there were qubits in the circuit that otherwise would have no gates acting on them. Which for example can occur in readout error calibration.